### PR TITLE
Move to GitHub, add autopkgtests, release 5.4

### PR DESCRIPTION
**Pull request by geofft**
This PR creates version 5.4 which
 * makes the examples actually build
 * adds an autopkgtest to build all the examples
 * adds Travis config from http://travis.debian.net/ to run autopkgtests
 * updates the URLs to GitHub
 * adds a README

### CODE REVIEW

**Comment by vulpicastor** [comment]
LGTM

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+sudo: required
+language: generic
+
+services:
+  - docker
+
+script:
+  - wget -O- http://travis.debian.net/script.sh | sh -
```
Comment (kaduk) [inline .travis.yml]: ```wget http |sh -```, such fun.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+tools (e.g., Ansible, Chef, Puppet, cfengine) that integrates cleanly
+with the Debian packaging system: you can apply configuration by
+installing a configuration package, and remove it cleanly by
+uninstalling it. config-package-dev uses `dpkg-divert` to inform the
+packaging system that it is changing files owned by another package,
+which means that your changes remain applied even when the original
+package is upgraded.
+
+Configuration packages are particularly useful for sites using Debian or
+Ubuntu (or another dpkg-based distribution) and are already running
```
Comment (kaduk) [inline README.md]: The "are" is not grammatical here.

File: examples/debhelper/debathena-transform-example-1.0/debian/compat [hunk 3]
```diff
@@ -1 +1 @@
-7
+9
```
Comment (kaduk) [inline examples/debhelper/debathena-transform-example-1.0/debian/compat]: I suppose that by bumping the compat level to 9 we potentially cause problems for people trying to use our latest stuff on older releases, but it's also not hard to work around.

Comment (geofft) [reply examples/debhelper/debathena-transform-example-1.0/debian/compat]: 9 is supported as far back as wheezy and trusty, so we're fine (continued users of squeeze or precise or something are on their own). I did mention this in the commit message but that's not super obvious from the PR interface.

File: .travis.yml (hunk 1 above)
Comment (geofft) [reply .travis.yml]: see the FAQ http://travis.debian.net/

File: README.md (hunk 2 above)
Comment (geofft) [reply README.md]: Fixed in HEAD, thanks.

**Pull request merged**